TypeSystem/Serialization/test.cc

```diff
@@ -60,6 +60,14 @@ CURRENT_STRUCT(Serializable) {
   bool operator<(const Serializable& rhs) const { return i < rhs.i; }
 };
 
+CURRENT_STRUCT(Int) {
+  CURRENT_FIELD(x, int32_t, 0);
```

Comment:
@sompylasar Why so serious? :) Your comments are really appreciated, and Dima being drunk doesn't mean he won't fix what should be fixed :)